test/dom/element.test.js

```diff
@@ -1,156 +1,183 @@
-'use strict';
+'use strict'
 
-var wows = require('vows');
-var DOMParser = require('../../lib/dom-parser').DOMParser;
+var wows = require('vows')
```

Comment:
Question: I was just surprised to still see those "imports", shouldn't they be gone from the test files by now?